.provisioners/_scripts/golang.sh

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env zsh
```

Comment:
this is a .sh file. Thus a bash file.
Maybe you meant to rename it to be .zsh ?